internals/jest/jest.config.js

```diff
@@ -0,0 +1,17 @@
+const logger = require('../../src/lib/logger');
+
+// jest.mock(logger);
+
+logger.info = () => {};
+
+const jestConfig = {
```

Comment:
This should be a json file, then in setup files make jest.setup.js file, there you can import mocks form __mocks__ folder, one of which is gonna be a logger.

Also it's better to use jest.mock('...', () => ) syntax, coz later in tests you will be able to do simple `jest.unmock`

Reply:
don't do assignments to noop, it will make things less obvious